### PR TITLE
feat(subscription-prompt): add dynamic pricing & discounts based on locale

### PR DESCRIPTION
**Pull request by keithamus**
Leaving this here as a 90% done "feature".

Will complete Monday

### CODE REVIEW

File: subscription-offer-prompt/index.js [hunk 1]
```diff
@@ -65,5 +66,23 @@ function createSubscriptionPrompt (html) {
 	};
 }
 
-export const initPrompt = executeWhen(createSubscriptionPrompt(createPopupHTML));
+function getSubscriptionPromptValues () {
+	const currencyCode = 'GBP';
```
Comment (keithamus) [inline subscription-offer-prompt/index.js]: This needs to be configured to the user's currency code. I don't know how to get or derive this. 

@matthew-andrews @wheresrhys @ironsidevsquincy @ifyio advice?

Comment (wheresrhys) [inline subscription-offer-prompt/index.js]: Don't think it's possible with ease at present. Once we ditch ie9 & 10 js support, and DAM gets deprecated, I would hope that membership provide us with a CORS enabled service for this sort of thing. As a stop gap you could add a json endpoint to product-selector to get this info

**Comment by keithamus** [comment]
@wheresrhys updated code. Could you please re-review?

**Comment by wheresrhys** [comment]
:+1:

**Pull request merged**